.github/workflows/publishVSCode.yml

```diff
@@ -66,14 +67,16 @@ jobs:
         with:
           node-version-file: '.nvmrc'
       - name: downloadExtensionsFromRelease
-        run: | 
+        run: |
           mkdir ./extensions
           gh release download v${{ env.PUBLISH_VERSION }} -D ./extensions
       - name: Display downloaded vsix files
         run: ls -R ./extensions
       - run: npm ci
       - run: find ./extensions -type f -name "*.vsix" -exec npx vsce publish --pat ${{ env.VSCE_PERSONAL_ACCESS_TOKEN }} --packagePath {} \;
-      - run: echo "SUCCESSFULLY published"
+      - run: echo "SUCCESSFULLY published on Microsoft VS Code Marketplace"
+      - run: find ./extensions -type f -name "*.vsix" -exec npx ovsx publish --pat ${{ env.OVSX_PAT }} --packagePath {} \;
```

Comment:
The idea is not to repeat any process like build, test, downloading vsix. Once the vsixes are downloaded, first we publish to market place and then to open-vsx registry. If both the processes are successful then we get ctcCloseSuccess.